presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreen.kt

```diff
@@ -241,9 +344,24 @@ fun EventScreen(
             },
             isValidInput = false,
             onAddButtonClicked = { visitorEmail ->
-                eventScreenEvent(EventScreenEvent.CheckVisitorExists(visitorEmail))
+
+                /** TODO Remove this logic for checking attendees already added to the viewModel */
```

Comment:
I was going to say exactly this

Just have an "AddVisitor" and have the ViewModel do all the checks for if the visitor was already added or not, and if it's a duplicate, etc